.github/workflows/gofmt.yml

```
@@ -0,0 +1,29 @@
+jobs:
+    test:
+        strategy:
+            matrix:
+                go-version: [1.24.x]
+                os: [ubuntu-latest]
+
+        runs-on: ${{ matrix.os }}
+
+        steps:
+            - name: Install Go
+              uses: actions/setup-go@v2
```

actionlint (1.7.4)

the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)